datasets/thelook_ecommerce/pipelines/_images/run_thelook_kub/fake.py

```diff
@@ -162,7 +162,7 @@ def main(
 
     # generate ghost events
     logging.info("generating ghost events")
-    for user_num in range(int(num_of_users)):
+    for user_num in range(int(num_of_users) * 5):
```

Comment:
Instead of multiplying by 5, we can control this via the parameters injected into the script and call this `num_of_ghost_events`